### PR TITLE
Lambda Function - Standard Fields (SX-1697)

### PR DESCRIPTION
**Pull request by etspaceman**
### Jira Ticket [SX-1697](https://singlecomm.atlassian.net/browse/SX-1697)
- [x] Ticket is linked in the title, ie `The Jira Ticket Title (SX-1697)`

###  Acceptance Criteria
- [x] Code Documentation (TSDoc and inline comments)
- [x] Unit Tests (wrote new tests and current tests passing)
- [x] Removed debug code
- [x] Inspected Code Diff
- [x] Added relevant QA information to Jira ticket

### An overview of the thinking process
I created the Lambda as a serverless service. The Jenkins pipelines haven't been tested yet because the dev branch is empty, but the triggers have been set up and I'd like to test that with this PR.

### Anything you think you might break with this PR
None

### Anything you’d like the reviewers opinion on specifically
This is my first NodeJS Lambda application so please be plentiful with the feadback.

### CODE REVIEW

File: Jenkinsfile-dev [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    }
+
+    stage('Integration Test'){ //tests the build
+          agent any
+
+          steps {
+            withCredentials(
+              [[$class: 'AmazonWebServicesCredentialsBinding',
+                accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                credentialsId: 'scripterStandardFieldService',
```
Comment (turnquistnt) [inline Jenkinsfile-dev]: Is this the right AWS credential? doesn't match the naming schemes of the other pipelines

Comment (turnquistnt) [reply Jenkinsfile-dev]: Also on line 105

Comment (etspaceman) [reply Jenkinsfile-dev]: Its not, good find. Ill fix

Comment (etspaceman) [reply Jenkinsfile-dev]: @turnquistnt this is fixed now.

File: microservices/scripter-standard-fields/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+plugins:
+  - serverless-offline
+
+package:
+  include:
+    - ../../node_modules/**
+
+functions:
+  get:
+    handler: handler.get
```
Comment (mikeblakeney) [inline microservices/scripter-standard-fields/serverless.yml]: We need to have one file per lambda/function.

Comment (etspaceman) [reply microservices/scripter-standard-fields/serverless.yml]: Just clarifying - 1 handler per function? 1 serverless.yml per function? Both?

**Pull request merged**